### DISCUSSION TITLE
Check if a block has a block tag

### DISCUSSION THREAD

**Discussion by Am-Gone** [Minestom/Minestom#1570] [Help] [answered]
How can I check if a block has a block tag like `#minecraft:climbable`? From what I've found minestom didn't implement it but the data generator already generates the json file for them in tags/block_tags.json

**Comment by mworzala** [comment, marked as answer]
It is probably available in `TagManager` (can get from `MinecraftServer`), but I am not at a computer to check.

**Comment by Am-Gone** [comment]
It works, thanks!
